components/locid/src/extensions/mod.rs

```diff
@@ -88,12 +87,17 @@ impl ExtensionType {
 
 /// A map of extensions associated with a given [`Locale`](crate::Locale).
 #[derive(Debug, Default, PartialEq, Eq, Clone, Hash, PartialOrd, Ord)]
-#[allow(missing_docs)] // TODO(#1028) - Add missing docs.
 #[non_exhaustive]
 pub struct Extensions {
+    /// A representation of the data for a Unicode extension subtag, when present in the locale identifer.
     pub unicode: Unicode,
+    /// A representation of the data for a transform extension subtag, when present in the locale identifer.
     pub transform: Transform,
+    /// A representation of the data for a private-use extension subtag, when present in the locale identifer.
     pub private: Private,
+    /// A sequence of any other extension subtags that are present in the locale identifier but are not formally
+    /// [defined](https://unicode.org/reports/tr35/) and represented explicitly as [`Unicode`], [`Transform`],
+    /// and [`Private`] are.
```

Comment:
Suggestion: don't say "subtag" here because it is confusing.

```suggestion
    /// A representation of the data for a Unicode extension, when present in the locale identifer.
    pub unicode: Unicode,
    /// A representation of the data for a transform extension, when present in the locale identifer.
    pub transform: Transform,
    /// A representation of the data for a private-use extension, when present in the locale identifer.
    pub private: Private,
    /// A sequence of any other extensions that are present in the locale identifier but are not formally
    /// [defined](https://unicode.org/reports/tr35/) and represented explicitly as [`Unicode`], [`Transform`],
    /// and [`Private`] are.
```